### DISCUSSION TITLE
[Help] How to open notes??

### DISCUSSION THREAD

**Discussion by vinoff** [zk-org/zk#352] [Help] [answered]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

So, I have set `editor = "neovide"` in the [tool] section of the config file, however, I am still unable to open notes in an external editor. I have tried with other editors but it doesn´t work.

What I do is:

 - zk list --interactive
 - Choose a random note
 - Press Enter.

It tells me "found 1 note" and outputs some of its content to the terminal. It doesn´t, however, open the note in the editor, which is what I want..

What am I doing wrong?

**Comment by mickael-menu** [comment, marked as answer]
You need to use `zk edit --interactive` instead. `zk list` is used to list the content of notes on stdout.

**Reply by vinoff** [reply]
Ahhhh! Thank you!
